### PR TITLE
Newhairjack

### PR DESCRIPTION
**Pull request by UncleBourbon**
## About The Pull Request
add new hairstyle: Jack

## Why It's Good For The Game
More customization

## Changelog
:cl:
add: Added new hairstyle: Jack
/:cl:

### CODE REVIEW

File: modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+//////////////////////////////////////
```
Comment (Cw3040) [inline modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:1]: the box is unnecessary, volra is big dumb, just do a normal comment on the same line as the datum

File: modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+//lumossnowflakehair hair hair hair //
+//////////////////////////////////////
```
Comment (VolraSlava) [inline modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:3]: come on bruh I told you to change this to something that isn't this ;_;

File: modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm (hunk 1 above)
Comment (UncleBourbon) [reply modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:1]: i kinda like the box tho its quirky

File: modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm (hunk 2 above)
Comment (UncleBourbon) [reply modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:3]: no

Comment (coldud13) [reply modular_lumos/code/modules/mob/dead/new_player/sprite_accessories/hair_head.dm:3]: yes please make the box less memey

**Review by coldud13 — APPROVED** [review]
ery noice

**Pull request merged**